package.json

```diff
@@ -2,6 +2,10 @@
   "name": "swaggerize-express",
   "version": "4.0.2",
   "author": "Trevor Livingston <trlivingston@paypal.com>",
+  "contributors": [
```

Comment:
I haven't been tracking contributors so if we're going to add this we need to add all of them.